extension/scripts/content/travelagency/ttTravelAgency.js

```diff
@@ -1,24 +1,41 @@
 "use strict";
 
-(() => {
-	for (const type of document.findAll(".travel-agency .torn-tabs > li:not([data-state='disabled'])")) {
-		type.addEventListener("click", async () => {
-			triggerCustomListener(EVENT_CHANNELS.TRAVEL_SELECT_TYPE, { type: type.find(".travel-name").innerText.toLowerCase() });
+(async () => {
+	if (await checkMobile() || await checkTablet()) {
+		document.find(".travel-agency").addEventListener("click", () => {
+			if (
+				event.target.closest(".travel-info-table-list[aria-selected*='true'][role*='tab']")
+			) {
+				triggerCustomListener(EVENT_CHANNELS.TRAVEL_SELECT_COUNTRY, null);

```

Comment:
If there are no details to pass through, simply skip this parameter and just pass 1 instead of the 2nd one being `null`.

Reply:
And you should also include the country if possible, as it's passed in the other trigger so not providing it could cause bugs.